docs/spec_draft.md

```diff
@@ -680,6 +681,48 @@ IEEE-754 specification.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.if
+
+### Semantics
+
+Returns the result of `true_branch` if `pred` is `true`, `false_branch` if
+`pred` is `false`.
```

Comment:
How about "Produces the output of"?
  * "Produces" follows the convention that other ops use and avoids ambiguity with `return`.
  * "the output of" matches how functions are defined in terms of inputs and outputs in the spec.

Reply:
Produces the output from executing exactly one branch from `true_branch` or
`false_branch` depending on the value of `pred`. Formally, if `pred` is `true`,
output of `true_branch` is returned, else if pred is `false`, output of
`false_branch` is returned.

Split it into two lines, still using the word `return` in second line.